lumicks/pylake/kymotracker/kymoline.py

```diff
@@ -319,8 +320,8 @@ def plot_msd(self, max_lag=None, **kwargs):
         plt.xlabel("Lag time [s]")
         plt.ylabel(f"Mean Squared Displacement [{self._kymo._calibration.unit_label}$^2$]")
 
-    def estimate_diffusion_ols(self, max_lag=None):
-        """Perform an unweighted fit to the MSD estimates to obtain a diffusion constant.
+    def estimate_diffusion(self, max_lag=None):
```

Comment:
as in the above comment, I think it'd be good to add the `method` argument from your next PR so that we don't immediately break this feature after release